frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -204,11 +204,8 @@ export function FunnelSingleStepState({ actionable = true }: FunnelSingleStepSta
 
     return (
         <div className="insight-empty-state funnels-empty-state">
-            <div className="empty-state-inner">
-                <div className="illustration-main">
-                    <PlusCircleOutlined />
-                </div>
-                <h2 className="text-xl leading-tight funnels-empty-state__title">Add another step!</h2>
+            <div className="empty-state-inner py-4">
```

Comment:
Actually, why this global change?

Reply:
huh, so strange, the empty state looked different yesterday (including the snapshot). The icon was misaligned to the top left and very small, it just seemed out of place, so I removed it completely. Now it's somehow fixed, I see it should be in the middle and it looks right 🤷‍♂️ I'll revert this.